php/containers.json

```diff
@@ -499,10 +499,18 @@
       ],
       "internal_port": "9200",
       "environment": [
+        "POSTGRES_HOST=nextcloud-aio-database",
         "TZ=%TIMEZONE%",
+        "ES_JAVA_OPTS=-Xms1024M -Xmx1024M -Des.enforce.bootstrap.checks=true",
```

Comment:
Check if we can change the loglevel to warn: https://www.elastic.co/guide/en/elasticsearch/reference/current/logging.html#configuring-logging-levels